
syntax = "proto3";

package cogment_app;

message EnvConfig {
}

message TrialConfig {
	EnvConfig env_config = 1;
}

message Observation {}

message Reward {}

message ClientAction {}

message TimeAction {}

message AsyncMessage {}
